src/components/expanded-state/chart/ChartExpandedStateHeader.js

```diff
@@ -126,12 +126,14 @@ export default function ChartExpandedStateHeader({
           <RowWithMargins
             height={30}
             justify="space-between"
+            marginHorizontal={1}
             marginVertical={android ? 4 : 1}
           >
             <ChartHeaderSubtitle
               color={
                 isNoPriceData ? colors.alpha(colors.blueGreyDark, 0.8) : color
               }
+              style={{ width: '67%' }}
```

Comment:
maybe you can use flex here? I am not a fan of magic %-s and I think this case is exactly what can be achieved with `{ flex: 1 }` and `{ flex: 2 }`